index.html

```diff
@@ -1,32 +1,164 @@
+                 <ul class="language-container">
+                   <li class="language html">HTML</li>
+                   <li class="language css">CSS</li>
+                   <li class="language java">Javascript</li>
+                 </ul>
+                 <button class="card-btn">See project</button>
+           </div>
+    </section>
+    <section class="contact-container">
+        <div class="one">
+         <h3 class="about">About Myself</h3>
+         <p class="bio">Hello I’m a software developer!
+             I can help you build a product , feature or website. 
+             Look through some of my work and experience! 
+             If you like what you see and have a project you need coded, don’t hesitate 
+             to contact me.
+         </p>
+         <h4 class="connect"><a class="connect-bottom" href="#">LET’S CONNECT</a></h4>
+           <ul class="icon bicon">
+             <li class="github"><a href="#"><img src="/img/github.svg" alt="github"></a></li>   
+             <li class="linkdin"><a href="#"><img src="/img/Linkedin icon.svg" alt="linkedin"></a></li>
+             <li class="v"><a href="#"><img src="/img/v.svg" alt="v"></a></li>
+             <li class="twiter"><a href="#"><img src="/img/twiter.svg" alt="twiter"></a></li>
+             <li class="m"><a href="#"><img src="/img/m.svg" alt="m"></a></li>
+             
+             
+         </ul>
+         <button class="card-btn resume">Get my resume</button>
+        </div>
+        <div class="two">
+            <div class="title-language">
+                <h2 class="tl">Language</h2>
+                <img src="/img/Icon - Chevron - Down - Larger.svg" alt="" class="dropdow">
+            </div>
+            <ul class="fotter-java">
+              <li class="space"><img src="/img/java.svg" alt="Javascript" class="language-pic"></li>
+              <li class="space"><h3 class="lp-title">Javascript</h3></li>
+            </ul>
+            <ul class="fotter-html">
+              <li class="space"><img src="/img/html.svg" alt="html" class="language-pic"></li>
+              <li class="space"><h3 class="lp-title">HTML</h3></li>
+            </ul>
+            <ul class="fotter-css">
+              <li class="space"><img src="/img/css.svg" alt="css" class="language-pic"></li>
+              <li class="space"><h3 class="lp-title">CSS</h3></li>
+            </ul>
+            <ul class="frameworks-container">
+              <li class="framework">Frameworks</li>
+              <li class="right-arow"><img src="/img/Disabled.svg" alt="right-arow" class="arow"></li>
+            </ul>
+            <hr>
+            <ul class="skills-container">
+              <li class="skills">Skills</li>
+              <li class="right-arow"><img src="/img/Disabled.svg" alt="right-arow" class="arow"></li>
+            </ul>
+            <hr>
+            
+        </div>
```

Comment:
- [ ] As requested by the previous reviewer, kindly use `ul` for the technology categories ("Languages", "Frameworks" and "Skills") , each of them in a `li`